app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -95,7 +101,7 @@ fun TestFlipCard(cardFace: CardFace, onClickToList: () -> Unit, id: String) {
             coroutineScope {
                 launch {
                     animate(
-                        initialValue = 1f, targetValue = 1.7f, animationSpec = tween(
+                        initialValue = 1f, targetValue = 1.3f, animationSpec = tween(
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)